_dashboards/admin-ui-index/snapshot-management.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: default
+title: Snapshot Management
+parent: Admin UI for index operations in OpenSearch Dashboards
+nav_order: 20
+---
+
+# Snapshot Management
+Introduced 2.5
+{: .label .label-purple }
+
+The Snapshot Management section in the admin UI allows you to preform operations available in the [Snapshots API]({{site.url}}{{site.baseurl}}/api-reference/snapshots/index/) from OpenSearch Dashboards' web UI.
+
+## Snapshot Policies
+
+[Snapshot Management]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-management) (SM) lets you automate taking snapshots. To use this feature, you need to install the [Index Management (IM) Plugin]({{site.url}}{{site.baseurl}}/im-plugin/index/). Snapshots store only incremental changes since the last snapshot. Thus, while taking an initial snapshot may be a heavy operation, subsequent snapshots have minimal overhead. To set up automatic snapshots, you have to create a snapshot policy with a desired SM schedule and configuration.
```

Comment:
We also have a snapshot management page in dashboards here: https://opensearch.org/docs/latest/opensearch/snapshots/sm-dashboards/. I think we should consolidate these. If we decide to keep the original, it should move to this section and this document can link to it. Otherwise, we can consolidate them in one document to avoid duplication.

Reply:
Also, it's not necessary to mention that they need to install the IM plugin. I think it comes with default installation now

Reply:
I will review the final iteration of this file after the other one is consolidated into it.

Reply:
I'll review this file when upon resolution of the above comments.

Reply:
I've moved the page over and made some updates and am currently working on doc review.